tensorflow_datasets/url_checksums/snli.txt

```diff
@@ -0,0 +1 @@
+https://nlp.stanford.edu/projects/snli/snli_1.0.zip 94550081 afb3d70a5af5d8de0d9d81e2637e0fb8c22d1235c2749d83125ca43dab0dbd3e
```

Comment:
Why put checksum in a different file. If I'm not wrong, checksum is already being retrieved and checked while it is being downloaded.

Reply:
This file is automatically generated when adding a new dataset.
When users `.download_and_prepare()` a datasets, the urls checksums are compared with this file to ensure the downloaded files are correct.

Reply:
@us In one of the previous PR, this has been added by @pierrot0 to store the checksum in a different file. Link is [Here](https://github.com/tensorflow/datasets/pull/247)

Reply:
@ParthS007 okay I missed it thank you!